website/docs/patterns/multi-select-patterns/partials/interaction-concepts/interaction-concepts.md

```diff
@@ -0,0 +1,52 @@
+When combining a Multi-select pattern with filtering, sorting, or pagination, you may encounter complex scenarios regarding how selected results should be persisted across pages and what the user expects when traversing large data sets.
+
+These interaction guidelines are not fully comprehensive. They should instead be considered as conversation starters between design and engineering on how best to optimize the user experience.
+
+!!! Info
+
+If you run into unique scenarios that these guidelines don’t account for, we would love to hear about them through our normal [support](/about/support) channels.
+!!!
+
+## Consistent communication
+
+The most important method to ensure users are interacting with the results they intend to is to clearly communicate what they have selected, or even that a selection of results exists, **at all times**.
+
+This can help to prevent unnecessary deletions of results or making unintended changes if what is currently selected is unclear.
+
+## Persisting the selection
+
+One of the most common questions around Multi-select patterns stems from when and how selected results are persisted when a user performs different actions on a Table. Consider the following scenario:
+
+A user is viewing a data set consisting of 10,000 results that is presented in a Table. In order to make changes to specific results the user may take the following actions:
+
+1. **Filter** the Table to display only a subset of the results.
+2. **Sort** the Table by a specific parameter to order the results in a meaningful way.
+3. Traverse through multiple pages of the Table through **Pagination**.
+
+How the selection is persisted across all of these actions can be confusing and open the user up to making unintended changes if what they have selected is persisted (or not persisted) in a logical manner.
```

Comment:
+1 to what @heatherlarsen mentioned. Maybe adding in an "end-user" would clarify this a bit more. 
```suggestion
How the selection is persisted across all of these actions can be confusing and open the end-user up to making unintended changes if what they have selected is persisted (or not persisted) in a logical manner.
```